recipes/barrnap/0.3/build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-mkdir -p $PREFIX
-mv ./* $PREFIX/
+mkdir -p $PREFIX/bin
+mv ./* $PREFIX/bin
```

Comment:
@johanneskoester what is the gain from this. You add a new layer of complexity to the tool that can have unwanted side effects - and you can not be sure that your test will capture it. Is there any big advantage of this?
I looked at the perl code and it looks for the current working dir. Inital tests with symlinks do not work. Is it really worth the pain?